public/app/core/components/RolePicker/ValueContainer.tsx

```diff
@@ -8,15 +8,17 @@ export interface Props {
   children: ReactNode;
   iconName?: IconName;
 }
-export const ValueContainer = ({ children, iconName }: Props) => {
+export const ValueContainer = forwardRef<HTMLDivElement, Props>(({ children, iconName }, ref) => {
```

Comment:
@Clarity-89 what is a forwardRef? interesting

Reply:
There's some good info in react docs: https://reactjs.org/docs/forwarding-refs.html